### DISCUSSION TITLE
tokens

### DISCUSSION THREAD

**Discussion by mqqqxon** [Acly/krita-ai-diffusion#1277] [Q&A]
Hello, I really like the plugin, but what should I do if I run out of tokens?

**Comment by Eggrollah** [comment]
> Hello, I really like the plugin, but what should I do if I run out of tokens?

I’m also wondering the same, I don’t have enough for generative fill but I’m not sure how to get more

**Comment by Acly** [comment]
You can buy more on the website: https://www.interstice.cloud/user

(I guess there is a missed opportunity here to put a button in the UI)

If you have a powerful GPU you can also install locally (from the Connection settings)
